mesh/mesh.cpp

```diff
@@ -1345,9 +1345,9 @@ void Mesh::FinalizeQuadMesh(int generate_edges, int refine,
 
 
 #ifdef MFEM_USE_GECKO
-void Mesh::GetGeckoElementReordering(Array<int> &ordering,
-                                     int iterations, int window,
-                                     int period, int seed)
+void Mesh::GetGeckoElementOrdering(Array<int> &ordering,
```

Comment:
Do not rename things that are already in the library -- I think you should revert this change.

Reply:
I generally agree, but I think the change is acceptable and desirable in this particular case

Reply:
Sorry for the late reply. I agree we shouldn't just change things in the API, but this case I felt was special. Since the library was never published, this code was hardly ever used by anyone outside the core team. That's why I thought the change was acceptable in this case.